src/shared/ui/section-label/section-label.tsx

```diff
@@ -1,9 +1,49 @@
-import './section-label.scss';
+import { HTMLAttributes } from 'react';
+import { type VariantProps, cva } from 'class-variance-authority';
+import classNames from 'classnames/bind';
 
-type SectionLabelProps = {
-  label: string;
-};
+import styles from './section-label.module.scss';
+
+type SectionLabelProps = HTMLAttributes<HTMLDivElement> &
+  VariantProps<typeof sectionLabelVariants> & { label: string };
+
+export const cx = classNames.bind(styles);
 
-export const SectionLabel = ({ label }: SectionLabelProps) => (
-  <div className="section-label">{label}</div>
-);
+const sectionLabelVariants = cva(cx('label'), {
```

Comment:
It seems to me that this component is the same size and font size everywhere. Why do we need variations?

Reply:
In our Figma layout, I was guided by it, fonts for the section-label are presented in two different sizes - 12 (small) and 14 (large) px. Should I remove this variability?

Reply:
> It seems to me that this component is the same size and font size everywhere. Why do we need variations?

In our Figma layout, I was guided by it, fonts for the section-label are presented in two different sizes - 12 (small) and 14 (large) px. Should I remove this variability?